<launch>
    <!-- Launch RealSense Node -->
    <include file="$(find realsense2_camera)/launch/rs_camera.launch" />
    <!-- AprilTag detector node -->
    
In this launch file, replace your_camera_package and your_camera_launch_file.launch with the appropriate package and laun
    <!-- Launch Object Tracker Node -->
    <node name="dog_position_tracker" pkg="object_tracker" type="cv_test.py" output="screen" />
</launch>